src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -164,6 +165,7 @@ public partial struct AssemblyReference
         public System.Reflection.Metadata.StringHandle Name { get { throw null; } }
         public System.Reflection.Metadata.BlobHandle PublicKeyOrToken { get { throw null; } }
         public System.Version Version { get { throw null; } }
+        public System.Reflection.AssemblyName AssemblyName { get { throw null; } }
```

Comment:
This should instead be a method and not a property as that was the approved API. 

```cs
public System.Reflection.AssemblyName GetAssemblyName()
```